examples/gno.land/r/gnoland/blog/admin.gno

```diff
@@ -4,38 +4,44 @@ import (
 	"std"
 	"strings"
 
-	"gno.land/p/demo/avl"
+	"gno.land/p/demo/access"
```

Comment:
Update the dependencies in `gno.mod` for `gno.land/r/gnoland/blog` refecting the same.

Reply:
`gnoland start` is not falling even though we have not updated the dependencies :/
Because of alphabetical order, maybe `gno.land/p/demo/access` loads before `gno.land/r/gnoland/blog`. That's why it's not complaining.

Reply:
I suggest making `gno lint` aware of this, since it's used with the CI.